Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Container for building Go binary.
 FROM golang:1.19.5-bullseye AS builder
 # Install dependencies
-RUN apt-get update && apt-get install -y build-essential git
+RUN apt-get update && apt-get install -y build-essential git ca-certificates
```

Comment:
do we need it here as well?

Reply:
probably not, the `golang` base image will have it embedded, removing it